jobs/e2e-test.groovy

```diff
@@ -176,8 +181,19 @@ def runLambdaTest() {
    // in the LambdaTest build.
    def e2eEnv = E2E_URL == "https://www.khanacademy.org" ? "prod" : "preprod";
 
+   // Determine which Fastly service to route requests to.
+   def fastlyServiceID = "" // Default to PROD [VCL].
+   if (params.FASTLY_SERVICE == "PROD [COMPUTE]") {
+      fastlyServiceID = "XVn2m1dzuB3ebdNAY6UsY1"
+   } else if (params.FASTLY_SERVICE == "STAGING [COMPUTE]") {
+      fastlyServiceID = "luUUdGK4AEAIz1vqRyQ180"
+   } else if (params.FASTLY_SERVICE == "TEST [COMPUTE]") {
+      fastlyServiceID = "uz47U4v9JmAHQhlAhnsHr4"
```

Comment:
I think we want to cookie values rather than the service IDs.  We are setting the cookie values in this PR https://github.com/Khan/webapp/pull/27243/files.

These are the cookie values:

```
Fastly Compute test: ZTM3YmE2YTFjNDZkZjEwMDYxMTM3NzQyM2VlYjgw. (service id uz47U4v9JmAHQhlAhnsHr4). aka www-accents.khanacademy.org

Fastly compute prod: NmU3ZDYxMmJjNDRkOGMwNDIzODg4ZDkyYTcxZTA4. (service id XVn2m1dzuB3ebdNAY6UsY1).

Fastly compute staging: ODI5NTIyOGIxZTdmODUxOTEwZjY0ZDM0NjdlYjJi. (service id luUUdGK4AEAIz1vqRyQ180).
```

Reply:
Thanks for catching this.  I've fixed this in https://github.com/Khan/jenkins-jobs/pull/251.